src/s1/t5/n1/exercici5/FileUtils.java

```diff
@@ -0,0 +1,35 @@
+package s1.t5.n1.exercici5;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+
+import s1.t5.n1.exercici4.DirectoryUtils;
+
+public class FileUtils extends DirectoryUtils {
+    public static void serializeObjectToFile(Serializable object, String filePath) throws IOException {
+        try (FileOutputStream fileOut = new FileOutputStream(filePath + ".ser");
+                ObjectOutputStream out = new ObjectOutputStream(fileOut);) {
+            out.writeObject(object);
+            System.out.println("Object has been successfully serialized to " + filePath);
+        } catch (IOException e) {
+            System.err.println("Error writing object to file.");
+        }
+    }
+
+    public static Serializable deserializeObjectFromFile(String filePath) throws IOException, ClassNotFoundException {
+        Serializable object = null;
```

Comment:
//You can reduce the code if you use try with resources you make sure that FileInputStream and ObjectInputStream are closed properly

public static Serializable deserializeObjectFromFile(String filePath) throws IOException, ClassNotFoundException {
    try (FileInputStream fileIn = new FileInputStream(filePath + ".ser");
         ObjectInputStream in = new ObjectInputStream(fileIn)) {
        return (Serializable) in.readObject();
    } catch (ClassCastException | IOException e) {
        System.err.println("Error leyendo el objeto del archivo: " + e.getMessage());
        return null;
    }
}